hw2/03-count.js

```diff
@@ -1,3 +1,28 @@
-// Add your code here
+const userInput = document.getElementById('userInput');
+const textContainer = document.querySelector('.textContainer');
 
-input.addEventListener('keydown', handleKeyDown);
+// Store the original text to reset when needed
+const originalText = textContainer.textContent;
+
+function handleKeyDown() {
+  setTimeout(() => {
```

Comment:
If you didn't want to do the setTimeout(), 0 way you could change from keydown to input in the event type. Your way works as well though.